tests/imfile-symlink-multi.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+# This is part of the rsyslog testbench, licensed under GPLv3
```

Comment:
sry, a final nit :-( could you license under ASL 2.0 if you have no concerns with that (if there are, keep as is!). We try to move toward ASL 2.0, but some old tests still bear GPLv3.